src/components/Author/Author.tsx

```diff
@@ -1,58 +1,44 @@
-import React, { Component, ReactNode, ReactElement } from 'react';
+import React, { FC, useContext } from 'react';
 
 import Avatar from '@material-ui/core/Avatar';
 import List from '@material-ui/core/List';
 
-import { DetailContextConsumer } from '../../pages/Version';
+import { DetailContext } from '../../pages/Version';
 import { Heading, AuthorListItem, AuthorListItemText } from './styles';
 import { isEmail } from '../../utils/url';
 
-class Authors extends Component {
-  public render(): ReactElement<HTMLElement> {
-    return (
-      <DetailContextConsumer>
-        {context => {
-          const { packageMeta } = context;
-
-          if (!packageMeta) {
-            return null;
-          }
-
-          return this.renderAuthor(packageMeta);
-        }}
-      </DetailContextConsumer>
-    );
+const Authors: FC = () => {
+  const { packageMeta } = useContext(DetailContext);
+
+  if (!packageMeta) {
+    return null;
   }
 
-  public renderLinkForMail(email: string, avatarComponent: ReactNode, packageName: string, version: string): ReactElement<HTMLElement> | ReactNode {
-    if (!email || isEmail(email) === false) {
-      return avatarComponent;
-    }
+  const { author, name: packageName, version } = packageMeta.latest;
 
-    return (
-      <a href={`mailto:${email}?subject=${packageName}@${version}`} target={'_top'}>
-        {avatarComponent}
-      </a>
-    );
+  if (!author) {
+    return null;
   }
 
-  public renderAuthor = ({ latest }) => {
```

Comment:
`latest` was implicitly typed as `any` before - this method wasn't type safe and was accessing properties that weren't defined on the type of `packageMeta`